core/src/io/anuke/mindustry/game/Rules.java

```diff
@@ -2,6 +2,11 @@
 
 import io.anuke.annotations.Annotations.Serialize;
 import io.anuke.arc.collection.Array;
+import io.anuke.arc.function.BooleanConsumer;
+import io.anuke.arc.function.BooleanProvider;
+import io.anuke.arc.function.Function;
+
+import java.util.function.Predicate;
```

Comment:
Why are these extra imports here? They're not used in the class. It's also worth mentioning that anything out of `java.util.function` isn't supported on Android; avoid it.

Reply:
hmm... I just forgot to cleanup imports